src/assets/i18n/pt-BR.json5

```diff
@@ -7925,28 +7341,23 @@
   "workspace-item.view.breadcrumbs": "Workspace View",
 
   // "workspace-item.view.title": "Workspace View",
-  // TODO New key - Add a translation
-  "workspace-item.view.title": "Workspace View",
+  "workspace-item.view.title": "Visão do Workspace",
```

Comment:
`Perspectiva do Workspace`

Reply:
I have doubt about the better translate in Brazilian Portuguese